### ISSUE TITLE
Issues when installing w/ pnpm

### ISSUE THREAD

**Issue by Uninen** [serebrov/emoji-mart-vue#236]
Using latest Vue and pnpm, adding latest version of emoji-mart-vue-fast results in following peer deps issues:

```
 WARN  Issues with peer dependencies found
.
└─┬ emoji-mart-vue-fast 11.2.0
  └─┬ vue-virtual-scroller 1.0.10
    ├── ✕ unmet peer vue@^2.6.11: found 3.2.40
    └─┬ vue-resize 0.4.5
      └── ✕ unmet peer vue@^2.3.0: found 3.2.40
```

**Comment by Q16solver** [comment]
Yeah.. I think it's an issue with the packages between the vue2 and vue3 versions not being separated properly, I think we should probably just have a new package called emoji-mart-vue3 or something similar, and it should only have dependencies for that

**Comment by serebrov** [comment]
The `vue-virtual-scroller` package is needed for the alternative picker implementation, `VirtualScrollPicker` which was a default implementation before, but it was more a problem-maker than an improvement. 
So it was replaced by a non-virtual scroll Implementation by default and left just for the case.

I think it should be OK to remove it and release a new major version of the package indicating the breaking change. If someone still uses it, there are several options:

* Stick to an older package version where the `VirtualScrollPicker` is still present
* Copy the `VirtualScrollPicker` into the project source, basing it on other code and components in this package
* If there is high demand, I could release it as a separate package (either build on top of this one or as a separate one)

**Comment by serebrov** [comment]
I published version `12.0.0`, it removes the `VirtualScrollPicker` and, as a consequence, a dependency on `vue-virtual-scroller` package.

**Comment by Q16solver** [comment]
@serebrov thats amazing work, thanks so much!

**Comment by Q16solver** [comment]
There are no longer any peer dependencies from my side when installing with npm, I would assume so as well with pnpm, so we should be able to safely close this issue now

**Comment by Uninen** [comment]
Fixed in #241
